docs/development/config-files.md

```diff
@@ -195,19 +195,26 @@ Some devices spam the network with hundreds of invalid `ConfigurationCC::InfoRep
 In order to get your configuration file included in this library, two things must be done:
 
 1. Update the index file using `yarn run config import -i`.
-1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
-1. Create a PR.
+2. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
+3. Create a PR.
```

Comment:
FYI, if you leave it at 1./1./1., you don't have to update the numbers if something gets changed ;)

Reply:
Didn't know :open_mouth: